### ISSUE TITLE
Minor visual issue in unregisterd version

### ISSUE THREAD

**Issue by gh658804** [RealRaven2000/QuickFolders#453]
There is a minor issue in the unregistered version when performing a mouseover the "renew-license" or "important message" button. Apart from the color-changing (which is fine), the button seems to enlarge by a couple of pixels which causes the toolbar and the window below it to move slightly. This is rather annoying and IMO unnecessary. Please see the attached image.

**Comment by RealRaven2000** [comment]
Could you paste the screenshot here on the Github website, it is missing?

I tried reproducing the issue by using an expired license (as you wrote something about "renew-license") but it is only a color change for me. The font size seems to remain the same, at 1.1em resulting in 13.2px screen size:

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/e49f0988-4dc6-4ba2-be3f-60285839d4b2)

this is the overriding CSS rule, which also applies on the :hover state:

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/c63f6266-f66b-450b-8c57-08c4168370a3)

maybe you can give some more context, when you said "unregistered version" - are you referring to a license key that is empty or expired?

**Comment by gh658804** [comment]
I tried earlier from Firefox but for some reason it didn't work. Here it is...
I'm using Windows 10 Pro...

![Issue](https://github.com/RealRaven2000/QuickFolders/assets/8936309/003d84e8-e440-4dd6-9630-275f6b174079)

EDIT: i tried pasting the attachment...

**Comment by RealRaven2000** [comment]
> I tried earlier from Firefox but for some reason it didn't work. Here it is... I'm using Windows 10 Pro...
> 

thanks, maybe that is a problem with inserting gifs, but it seems to be connected to showing the update notice as well... I am not sure whether you are aware (or whether I communicate this badly) but clicking the button will reset the button to its normal state. The idea behind this is that you should read the change log popup without forcing it on screen at a random time  (when Tb decides it). Again, for me it looks the same on hover (toolbar height does not change)

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/14d5ec01-55a6-41ed-a774-3f556c4d0d22)

**Comment by gh658804** [comment]
The theme i'm using is the default "System theme - auto". i tried playing with layout and density but nothing changed. I also disabled all other addons but again, no luck. Is there anything else that might affect the appearance of the button? My tabs look different than yours...

![Img](https://github.com/RealRaven2000/QuickFolders/assets/8936309/ae2a90aa-481b-41e4-8a7d-e7764f8316c6)

**Comment by gh658804** [comment]
OK, i found it. Options/Bling My Tabs/Apple pills. That's the cause...

All others are ok, in Apple Pills, the problem is visible.

**Comment by RealRaven2000** [comment]
Found the rule (applies to all tabs / buttons in the apple pills theme) : 
![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/5ecd2b86-486d-4405-866f-d9cf15f2b939)

Here is a test version with new rules - you have to restart Thunderbird to force reloading the style sheet:

[QuickFolders-mx-6.4.2pre4.zip](https://github.com/RealRaven2000/QuickFolders/files/14002236/QuickFolders-mx-6.4.2pre4.zip)

---

To install this version, download the zip file and drag it into Thunderbird Add-ons Manager without extracting contents. Restart Tb to force reloading cached style sheets.

**Comment by RealRaven2000** [comment]
Also added some color improvements for active tabs (and removed some of the obnoxious text shadows) in apple pills theme:

[QuickFolders-mx-6.4.2pre6.zip](https://github.com/RealRaven2000/QuickFolders/files/14002362/QuickFolders-mx-6.4.2pre6.zip)

---

To install this version, download the zip file and drag it into Thunderbird Add-ons Manager without extracting contents. Restart Tb to force reloading cached style sheets.

**Comment by gh658804** [comment]
Hello,
I updated and then directly tested the pre6 version and the problem was still there. It seems i had to restart TB in order for the pre6 version to be activated and after that, the issue disappeared.
I also noticed (which doesn't matter to me) that when i mouseover-ed the message, it seemed like 1 pixel was added vertically to the whole panel but that was it. It happened the first time i performed the mouseover and subsequent mouse-leave & enter did nothing, it worked without any change in the vertical height.

Anyway, the original problem does not exist anymore. Thank you very much for fixing for this annoyance.
Have a nice day.

**Comment by RealRaven2000** [comment]
> I updated and then directly tested the pre6 version and the problem was still there. It seems i had to restart TB in order for the pre6 version to be activated and after that, the issue disappeared.

that's correct, I noted this in the test version comments - style sheets (which do the layout of the user interface) are cached by Thunderbird and  only reloaded when we start it again. Hope the color choices for the text (current folder) work for you on apple.

**Comment by gh658804** [comment]
> Hope the color choices for the text (current folder) work for you on apple.

Yes, it looks nice, on Windows though... :) I just prefer it because it looks more elegant.

EDIT: Oh, you said the "current folder". Well, that's not so good. It's not easily readable...
